models/public/swin-tiny-patch4-window7-224/README.md

```diff
@@ -0,0 +1,84 @@
+# swin-tiny-patch4-window7-224
+
+## Use Case and High-Level Description
+
+The `swin-tiny-patch4-window7-224` model is a `tiny` version of the Swin Transformer image classification models pre-trained on ImageNet dataset. Swin Transformer is Hierarchical Vision Transformer whose representation is computed with shifted windows. Stages of tiny version of model have 2, 2, 6, 2 layers respectively. Number of channels of the hidden layers in the first stage for tiny variant is 96.
```

Comment:
what about adding information about components which added in model name (patch4, window7)? I think it can be useful if you provide what they means

Reply:
Added